docker/Dockerfile

```diff
@@ -6,6 +6,7 @@ RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/community" >>/etc/apk/reposi
 	apk add --update bash runit conntrack-tools iproute2 util-linux curl && \
 	rm -rf /var/cache/apk/*
 ADD ./docker.tgz /
+ADD ./ebpf.tgz /usr/libexec/scope/
```

Comment:
Is it worth the pain of compressing the file? And .. is it many files? (i.e. it may make things simpler to simply use the plain ebpf object file)